src/main/java/uk/gov/hmcts/darts/arm/component/impl/ArmResponseFilesProcessSingleElementImpl.java

```diff
@@ -544,6 +545,17 @@ private ObjectRecordStatusEnum updateExternalObjectDirectoryStatusAndVerificatio
         }
     }
 
+    private void updateTransferAttempts(ExternalObjectDirectoryEntity externalObjectDirectoryEntity) {
+        int currentNumberOfAttempts = externalObjectDirectoryEntity.getTransferAttempts();
+        log.debug(
+            "Updating failed transfer attempts from {} to {} for ID {}",
+            currentNumberOfAttempts,
+            currentNumberOfAttempts + 1,
+            externalObjectDirectoryEntity.getId()
+        );
+        externalObjectDirectoryEntity.setTransferAttempts(currentNumberOfAttempts + 1);
```

Comment:
does it need to be saved? or is it inside a @Transactional

Reply:
No, in the next method call the entity will be saved in one go.

Reply:
Not required, next method call does it.